src/property.ts

```diff
@@ -90,6 +94,11 @@ export const NESTED_ENCODING_PROPERTIES: NestedEncodingProperty[] = [
     property: Property.SCALE_TYPE,
     parent: 'scale',
     child: 'type'
+  },
+  {
+    property: Property.SCALE_ZERO,
```

Comment:
I would use `reduce` method to generate `NESTED_ENCODING_PROPERTIES` that would have the following output for now.  (Add a test!)

```
{
  bin: ['maxbins']
  scale: ['type', 'zero'] 
}
```

so that you can use it [here](https://github.com/uwdata/compassql/pull/113/files#diff-e723711227f2a0ee0232fccd5a5a1f35R273).  

Make sure that if you add another nested encoding property to 
`NESTED_ENCODING_PROPERTIES` you update this object for free.